Sources/DistributedSystem/ChannelHandler.swift

```diff
@@ -33,14 +37,18 @@ class ChannelHandler: ChannelInboundHandler {
 
     private var logger: Logger { actorSystem.logger }
 
-    private let id: UInt32
+    let id: UInt32
     private let actorSystem: DistributedSystem
     private let address: SocketAddress?
     private var writeBufferHighWatermark: Int
     private var targetFuncs = [String]()
 
     static let name = "channelHandler"
 
+    private func logPrefix(_ channel: Channel) -> String {
```

Comment:
Maybe use logger metadata to store channel description?

Reply:
May be it looks better from the API point of view, but in logs it is better to me the format I use now.